XenAdminTests/WizardTests/NewPolicyWizardTest.cs

```diff
@@ -38,9 +38,9 @@ namespace XenAdminTests.WizardTests.cowleyPolicies_xml
 {
 
     [TestFixture, Category(TestCategories.UICategoryB)]
-    class NewPolicyWizardTest : WizardTest<NewPolicyWizard>

```

Comment:
This won't solve the generics problem we have in the tests, because the NewPolicyWizard is generic and we need to use NewPolicyWizardSpecific<XenAPI.VMPP> instead. This would work:
 `class NewPolicyWizardTest : WizardTest<NewPolicyWizardSpecific<XenAPI.VMPP>>`